applications/CompressiblePotentialFlowApplication/custom_processes/move_model_part_process.h

```diff
@@ -0,0 +1,91 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:        BSD License
+//                  Kratos default license: kratos/license.txt
+//
+//  Main authors:    Marc Núñez,
+//
+
+#ifndef KRATOS_MOVE_MODEL_PART_PROCESS_H
+#define KRATOS_MOVE_MODEL_PART_PROCESS_H
+
+#include "includes/define.h"
+
+#include "includes/kratos_parameters.h"
+#include "includes/model_part.h"
+#include "processes/process.h"
+
+namespace Kratos
+{
+
+class MoveModelPartProcess: public Process
```

Comment:
I think throughout Kratos many of those exist, maybe eventually we can have this unified in the Core

I also have a version of this (works also in 3D), but only python, haven't done the effort yet to port it to Kratos...

Just saying that hopefully some day we will have a global solution :)